src/interaction/action/view/mousewheel-scroll.ts

```diff
@@ -0,0 +1,59 @@
+import { clamp, get, isObject, size, valuesOfKey } from "@antv/util";
+import { COMPONENT_TYPE } from "../../../constant";
+import { Action } from '..';
+import { LooseObject, ScrollbarOption } from "../../../interface";
+
+
+function isWheelDown(event: LooseObject) {
+  const wheelEvent = event.gEvent.originalEvent as WheelEvent;
+  return wheelEvent.deltaY > 0;
+}
+
+const DEFAULT_WHEELSPEED = 1;
+const DEFAULT_CFG = {
+  wheelSpeed: DEFAULT_WHEELSPEED
+};
+
+class MousewheelScroll extends Action {
+  protected wheelSpeed = DEFAULT_WHEELSPEED;
+
+  init() {
+    const mouseWheelCfg = this.getMousewheelCfg();
+    if (mouseWheelCfg) {
+      this.wheelSpeed = mouseWheelCfg.wheelSpeed || DEFAULT_WHEELSPEED;
+    }
+  }
+
+  private getMousewheelCfg()  {
+    const { view } = this.context;
+    const scrollbarCfg: ScrollbarOption =  view.getOptions().scrollbar;
+    return (isObject(scrollbarCfg) && isObject(scrollbarCfg.enableMouseWheel)) ?
+      scrollbarCfg.enableMouseWheel : DEFAULT_CFG;
+  }
+
+  public scroll() {
+    const { view, event } = this.context;
```

Comment:
#3639 provides setter and getter methods for scrollbar，Once it merges into the master branch, you can more easily set scrolling using the following method:
```ts
// get the orient of scrollbar
const { type } = scrollbarController.getValidScrollbarCfg();
const { deltaX, deltaY } = event.gEvent.originalEvent;
const wheelOffset = type === "horizontal" ? deltaX : deltaY;
const currentRatio = scrollbarController.getValue();
// using `wheelSpeedRatio` instead of `wheelSpeed` can sense the wheel speed.
scrollbarController.setValue(currentRatio + wheelSpeedRatio * wheelOffset);
```

Reply:
Ok, that's nice to have getter and setter methods.
Also thanks for your code fragment, I realized I forgot to test for different scrollbar types.
But I can't get your point with `wheelSpeedRatio`. Maybe it looks confusing, but I proposed a `wheel Speed` to be a number of chart's data items that being scrolled on a single mouse wheel scroll, so the bigger this value is, the faster you scroll through the chart.

Reply:
Well, `Wheel Speed` determines the number of data items to scroll through at a time is a good idea.